### ISSUE TITLE
A few errors with the new version...

### ISSUE THREAD

**Issue by mosanger** [valda/garybash#18]
I'm getting following error everytime i rebuild the bashed patch:

```
E:\Fallout 3\Mopy\bosh.py:566: DeprecationWarning: 'I' format requires 0 <= number <= 4294967295
def packSub(self,type,data,*values):
```

And the following only when i activate the new VATS-Playback-Delay Tweak:

```
Traceback (most recent call last):
  File "E:\Fallout 3\Mopy\basher.py", line 4924, in Execute
    raise
  File "E:\Fallout 3\Mopy\basher.py", line 4886, in Execute
    patchFile.buildPatch(log,SubProgress(progress,0.8,0.9))#no speeding needed/really possible (less than 1/4 second even with large LO)
  File "E:\Fallout 3\Mopy\bosh.py", line 15980, in buildPatch
    patcher.buildPatch(log,SubProgress(subProgress,index))
  File "E:\Fallout 3\Mopy\bosh.py", line 20120, in buildPatch
    tweak.buildPatch(self.patchFile,keep,log)
  File "E:\Fallout 3\Mopy\bosh.py", line 19844, in buildPatch
    fid = gmst.fid = keep(gmst.getFallout3Fid())
  File "E:\Fallout 3\Mopy\bosh.py", line 3002, in getFallout3Fid
    return (GPath('Fallout3.esm'), myClass.fallout3Ids[self.eid])
KeyError: '0.01'
```

**Comment by valda** [comment]
oops!
fixed at ff77d08179f624ce91eebb6cfbbc2e2dbce0b410

**Comment by valda** [comment]
done.
